libs/evcs-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -0,0 +1,499 @@
+            List.of(
+                    new EvcsCreateUserVCsDto(
+                            VcFixtures.VC_ADDRESS.getVcString(),
+                            EvcsVCState.CURRENT,
+                            Map.of(
+                                    "reason", "testing",
+                                    "txmaEventId", "txma-event-id-2",
+                                    "timestampMs", "1714478033959"),
+                            ONLINE));
+
+    private static final List<EvcsUpdateUserVCsDto> EVCS_UPDATE_USER_VCS_DTO =
+            List.of(
+                    new EvcsUpdateUserVCsDto(
+                            "VC_Signature1",
+                            EvcsVCState.HISTORIC,
+                            Map.of(
+                                    "reason", "testing",
+                                    "txmaEventId", "txma-event-id-2",
+                                    "timestampMs", "1714478033959")));
+
+    @Mock ConfigService mockConfigService;
+
+    @BeforeEach
+    void setUp(MockServer mockServer) {
+        when(mockConfigService.getParameter(ConfigurationVariable.EVCS_APPLICATION_URL))
+                .thenReturn("http://localhost:" + mockServer.getPort());
+        lenient()
+                .when(mockConfigService.getSecret(ConfigurationVariable.EVCS_API_KEY))
+                .thenReturn(EVCS_API_KEY);
+    }
+
+    @Pact(provider = "EvcsProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact validRetrieveVcRequestReturnsUsersVcWith200(
+            PactDslWithProvider builder) {
+        return builder.given("EVCS client exist")
+                .given("VCS exist")
+                .given("EVC has users VC")
+                .given("test-evcs-api-key is a valid API key")
+                .given("test-acess-token is a valid access token")
+                .given("test-user-id has one PENDING_RETURN VC")
+                .uponReceiving("A request to get users VCS")
+                .path("/vcs/" + TEST_USER_ID)
+                .query("state=" + PENDING_RETURN)
+                .method("GET")
+                .headers(
+                        Map.of(
+                                "x-api-key",
+                                EVCS_API_KEY,
+                                "Authorization",
+                                "Bearer " + TEST_EVCS_ACCESS_TOKEN))
+                .willRespondWith()
+                .status(200)
+                .body(getResponseBodyForUserVC())
+                .toPact();
+    }
+
+    private DslPart getResponseBodyForUserVC() {
+        return newJsonBody(
+                        body -> {
+                            body.array(
+                                    "vcs",
+                                    vcs -> {
+                                        vcs.object(
+                                                vc -> {
+                                                    vc.stringType("vc", "vc");
```

Comment:
You might need to make this at least a _plausible_ looking VC - EVCS is going to break this apart to extract the signature.

I suggest using a simple test VC value - no need to worry about the contents, but it'll need to be a valid JWT.